packages/@o3r/schematics/src/utility/environment.ts

```diff
@@ -20,7 +20,7 @@ export const getEnvironmentInfo = () => {
   const packageManagerInfo = getPackageManagerInfo();
   let otterCorePackageJsonPath: string | undefined;
   try {
-    otterCorePackageJsonPath = require.resolve('@o3r/core/package.json');
```

Comment:
Hmmmm, why?

Reply:
I forget to put it back :)
it was not working while testing with node-modules instead of pnp because nx see that as a circular dependencies between @o3r/schematics and @o3r/core